main.py

```diff
@@ -2,6 +2,35 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    Genre.objects.all().delete()
+    Actor.objects.all().delete()
+
+    Genre.objects.create(name = "Western",)
+    Genre.objects.create(name ="Action", )
+    Genre.objects.create(name = "Dramma", )
```

Comment:
There is a typo in the genre name 'Dramma'. It should be 'Drama'. This is corrected later in the script, but it's better to avoid the typo initially.